libs/core/src/lib/shellbar/user-menu/shellbar-user-menu.component.html

```diff
@@ -4,13 +4,15 @@
             class="fd-button fd-shellbar__button fd-shellbar__button--user-menu"
             [disabled]="disabled"
             [fdMenuTrigger]="menu"
+            [attr.aria-label]="user.ariaLabel"
         >
             <fd-avatar
                 size="xs"
                 *ngIf="!user.image"
                 colorAccent="{{ user.colorAccent }}"
                 [circle]="true"
                 label="{{ user.fullName }}"
+                [attr.alt]="user.alt"
```

Comment:
This is not read by the screenreader as the alt attribute is only intended for img elements. The parent element's aria-label is read instead (and in the example here, the text is the same)